staging/src/k8s.io/kube-scheduler/config/v1alpha1/types.go

```diff
@@ -135,8 +135,10 @@ type SchedulerPolicyConfigMapSource struct {
 type KubeSchedulerLeaderElectionConfiguration struct {
 	componentbaseconfigv1alpha1.LeaderElectionConfiguration `json:",inline"`
 	// LockObjectNamespace defines the namespace of the lock object
+	// DEPRECATED: will be removed in favor of LeaderElectionConfiguration.resourceNamespace
 	LockObjectNamespace string `json:"lockObjectNamespace"`
 	// LockObjectName defines the lock object name
+	// DEPRECATED: will be removed in favor of LeaderElectionConfiguration.resourceName
```

Comment:
LeaderElectionConfiguration is inlined, so is not visible when serialized. prefer `will be removed in favor of resourceName `